stock_landed_costs_auto_create/models/purchase_order.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    landed_costs_ids = fields.One2many(
```

Comment:
@victoralmau Shouldn't both behaviour separated in two modules ? Storing landed costs on purchase order and auto creation ? @pedrobaeza Could you update your review?

Reply:
ping @pedrobaeza